doc/examples/color_exposure/plot_local_equalize.py

```diff
@@ -34,27 +34,27 @@
 matplotlib.rcParams['font.size'] = 9
 
 
-def plot_img_and_hist(img, axes, bins=256):
+def plot_img_and_hist(image, axes, bins=256):
```

Comment:
@sciunto how about deprecating this in favour of `plot_image_and_histogram`? =) It's easy with our `deprecated` decorator!